docs/attributes-registry/build.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Build
+
+## Build Attributes
+
+This group describes attributes specific to builds (ci).
+
+| Attribute                      | Type   | Description                                  | Examples                                                           | Stability                                                        |
+| ------------------------------ | ------ | -------------------------------------------- | ------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| `build.build.artifact.id`      | string | The id of the build artifact.                | `120912`                                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
We have namespace `build.build`
In https://github.com/open-telemetry/semantic-conventions/issues/915#issuecomment-2115796005 this was just `build`.
eg.
```suggestion
| `build.artifact.id`      | string | The id of the build artifact.                | `120912`                                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Reply:
all md files are autogenerated. Any changes here should be fixed/made in corresponding yaml file